gammapy/astro/darkmatter/profiles.py

```diff
@@ -87,6 +88,62 @@ def integrate_spectrum_separation(self, func, xmin, xmax, separation, ndecade):
         return val.sum()
 
 
+class ZhaoProfile(DMProfile):
+    r"""Zaho Profile.
+    .. math::
```

Comment:
Add a small description here, to say that which equation this utilises. i.e.
`Equation 1 from Zhao (1996)`